test/GithubApi.Repositories.test.js

```diff
@@ -0,0 +1,81 @@
+const agent = require('superagent');
+const chai = require('chai');
+const md5 = require('md5');
+
+chai.use(require('chai-subset'));
+
+const { expect } = chai;
+
+const url = 'https://api.github.com';
+
+describe('Given a user the github', () => {
+  const username = 'aperdomob';
+
+  describe('when get user', () => {
+    let repositories;
+    let repository;
+    let user;
+    const repositorySearch = 'jasmine-awesome-report';
```

Comment:
i think that is correct, for example in the files of AgileTesting they declared the variables here.

![image](https://user-images.githubusercontent.com/38733280/102646605-58dd4b00-4132-11eb-88c8-b968f25deedb.png)

Because i use this variables in this context, not in the global :)

Reply:
yes to the scope! but at least repositorySearch should be in the describe before because it's data you need for the test, and that's something that should be define independently because it's a part of the preparation of data section.